.gitlab-ci.yml

```diff
@@ -767,14 +767,18 @@ test_full_integration:
     - sleep 10
     - export DOCKER_HOST="unix:///var/run/docker.sock"
     - docker version
-    - apk --update --no-cache add bash curl git openssh device-mapper py-pip
-      iptables gcc python2-dev libffi-dev libc-dev openssl-dev make python3
-      python3-dev e2fsprogs-extra openssl jq
+
+    # Get and install the integration test requirements from the integration repository
+    - wget https://github.com/mendersoftware/integration/blob/master/tests/requirements/apk-requirements.txt
```

Comment:
You don't need to download it, you have it locally at  `${CI_PROJECT_DIR}/../integration/requirements/apk-requirements.txt`. It is better to use the local one, so that it is checked out to the appropriate branch or PR.

Instead of using `${CI_PROJECT_DIR}/../`, just move the line: `- export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)` up here

Reply:
Same for python below

Reply:
Oh, the path is wrong, it is missing `/tests/`, but you get the point.

Reply:
I was a bit wrong with my guidance here. This should be done **after** we restore the workspace, which is done here: https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml#L779-L786

Sorry for that

Reply:
An now it works :raised_hands: The integration test failure is unrelated